appeals/api/src/server/endpoints/integrations/integrations.broadcasters/appeal.js

```diff
@@ -7,8 +8,58 @@ export const broadcastAppeal = async (
 	/** @type {string} */ updateType = EventType.Update
 ) => {
 	if (!config.serviceBusEnabled) {
-		return false;
+		//return false;
 	}
 
 	pino.info({ appealId, updateType });
+	// const appeal = await databaseConnector.appeal.findUnique({
```

Comment:
leftover?

Reply:
No I'm going to fast follow with https://pins-ds.atlassian.net/browse/ASI-17, that's the code that loads the appeal to be broadcast